src/context/AppProvider/AppContext.tsx

```diff
@@ -93,19 +93,18 @@ export const AppProvider = ({ children }: { children: React.ReactNode }) => {
   useEffect(() => {
     if (!wallet) return
     const walletSupportedChainIds = knownChainIds.filter(chainId => {
-      const chainAccountIds = accountIdsByChainId[chainId] ?? []
-      return walletSupportsChain({ chainId, wallet, isSnapInstalled, chainAccountIds })
+      return walletDeviceSupportsChain({ chainId, wallet, isSnapInstalled })
```

Comment:
This voids runtime support checks for Ledger, which have been moved to `walletSupportsChain`

Reply:
This should be correct, what we're looking for is setting the list of chains the device actually supports at a device level before considering the concept of a user connecting a chain. This is used to drive the list of chains available to connect when managing accounts.